src-tauri/risuko-webview-upgrade/android/src/main/java/app/risuko/webview_upgrade/WebViewUpgradeBootstrap.kt

```
@@ -0,0 +1,141 @@
+            }
+        } catch (t: Throwable) {
+            Log.e(LOG_TAG, "bootstrap failed", t)
+        }
+    }
+
+    /** @return true if the swap was applied. */
+    private fun trySwap(context: Context, packageName: String, fallbackMajor: Int, minSupported: Int): Boolean {
+        return try {
+            WebViewSwap.swapToInstalledPackage(context, packageName)
+            true
+        } catch (t: Throwable) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)